### PR TITLE
Create user name page in onboarding flow

### PR DESCRIPTION
**Pull request by curregoz**
Simulator Screenshot - iPhone 14 Pro Max - 2023-05-02 at 15 15 52

### CODE REVIEW

File: lib/app/onboarding/name/name_page.dart [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            ],
+          ),
+          Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              InputTextWd(
+                onChange: pageProvider.onNameChanged,
+                hintText: intl.onboardingNamePage_Dummy,
+                validate: [
+                  LengthLimitingTextInputFormatter(3),
```
Comment (kimberllyflorez) [inline lib/app/onboarding/name/name_page.dart:73]: why is the limit 3

File: lib/app/onboarding/welcome/welcome_provider.dart [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  Stream<RouteSpec> get routes => _routes;
+
+  void onContinuePressed() {
+    _routes.add(RouteSpec.push(route: const NameRoute()));
+  }
+
+  @override
+  void dispose() {
+    print('cosito');
```
Comment (kimberllyflorez) [inline lib/app/onboarding/welcome/welcome_provider.dart:17]: why do you need to print cosito here?

**Review by kimberllyflorez — APPROVED** [review]

File: lib/app/onboarding/welcome/welcome_provider.dart (hunk 2 above)
Comment (curregoz) [reply lib/app/onboarding/welcome/welcome_provider.dart:17]: ups!

File: lib/app/onboarding/name/name_page.dart (hunk 1 above)
Comment (curregoz) [reply lib/app/onboarding/name/name_page.dart:73]: ups!

**Pull request merged**